src/program.ts

```diff
@@ -2962,6 +2964,14 @@ export class File extends Element {
     if (!exports) this.exports = exports = new Map();
     exports.set(name, element);
     if (this.source.sourceKind == SourceKind.LIBRARY_ENTRY) this.program.ensureGlobal(name, element);
+
+    // Also, add to the namespaces that caputure our exports
+    for(let i = 0; i < this.aliasNamespaces.length; i++) {
+      let ns = this.aliasNamespaces[i];
+      if (!ns.lookup(name)) {
+        ns.add(name, element);
+      }
```

Comment:
Did you find that the additional check is necessary here? I guess it shouldn't, because `File#add` would already terminate if there is a duplicate, and nothing else should be adding to the alias namespaces? Or is there something?

Reply:
Oh no I just put it there because I thought I needed it :joy: I'll remove it :smile: